transport/src/main/java/io/netty/channel/AbstractChannelHandlerContext.java

```diff
@@ -35,6 +36,18 @@
     private final String name;
     private boolean removed;
 
+    // This will set to true once the ChannelHandler.handlerAdded(...) method was called. We need to keep track of this
+    // to ensure we will never call another ChannelHandler method before handlerAdded(...) was called to guard against
+    // ordering issues. handlerAdded(...) MUST be the first method that is called per handler when become part of the
```

Comment:
- per handler -> for handler
- when become part of -> when it becomes a part of